index.js

```diff
@@ -11,13 +13,134 @@ const FILE = 'pokedex.json';
 
 // Init express app
 const app = express();
+console.log("done creating app");
+
+// tell app to use the module
+app.use(express.json());
+app.use(express.urlencoded({
+    extended: true
+}));
+
+// method-override
+const methodOverride = require('method-override');
+app.use(methodOverride('_method'));
+
+// this line below, sets a layout look to your express project
+const reactEngine = require('express-react-views').createEngine();
+app.engine('jsx', reactEngine);
+
+// this tells express where to look for the view files
+app.set('views', __dirname + '/views');
+
+// this line sets react to be the default view engine
+app.set('view engine', 'jsx');
+
+// app.get('/', (req, res) => {
+//   // running this will let express to run home.handlebars file in your views folder
+//   res.render('home')
+// })
+
 
 /**
  * ===================================
  * Routes
  * ===================================
  */
 
+app.get('/', (request, response)=> {
+    const nameArr = [];
```

Comment:
```suggestion
    let nameArr = [];
```
has better semantic meaning since we are changing the array, although const should technically work